Gemfile.lock

```diff
@@ -394,6 +395,9 @@ GEM
       connection_pool (>= 2.2.2)
       rack (~> 2.0)
       redis (>= 4.2.0)
+    sidekiq-status (1.1.4)
```

Comment:
This does not seem maintained anymore. See the readme at https://github.com/utgarda/sidekiq-status. Have you considered using the fork instead?

Reply:
I remembered we discussed this and it slipped my mind to update, I will make the change to move over to the fork.

Reply:
Updated to the latest in https://github.com/SaraAlert/SaraAlert/pull/987/commits/5be75cbe252a41cd26747d9b35a057ef7a118ab7.